packages/web-components/src/divider/divider.ts

```diff
@@ -16,7 +23,7 @@ export class Divider extends FASTElement {
    * HTML Attribute: role
    */
   @attr
-  public role: DividerRole = DividerRole.separator;
+  public role!: DividerRole;
```

Comment:
Should `role` be optional since we probably don't want to require authors to define `role` all the time?

Reply:
I think it can move to optional, yes.

Reply:
Ah - this looks like a TS issue because the types conflict, so I think this is likely the needed approach, otherwise we end up typing as "string" which broadens the type to something we wouldn't want to support.